CHANGELOG.md

```diff
@@ -19,6 +19,8 @@ way to update this template, but currently, we follow a pattern:
   [#939](https://github.com/sharetribe/flex-template-web/pull/939)
 * [add] Added a small comment to documentation about the current state of code-splitting.
   [#938](https://github.com/sharetribe/flex-template-web/pull/938)
+* [remove] Removed plain text parts of email templates.
+  [#942](https://github.com/sharetribe/flex-template-web/pull/942)   
```

Comment:
@anttiSalmivaara we have kept this in chronological order, newest/current PR being the first bullet.